index.html

```diff
@@ -23,29 +23,227 @@
                 </a>
             </div>
         </nav>
-    </header>
-    <main class="main">
-        <section class="headline">
-            <h2 class="headline-text">I’m Belay <br> Glad to see <br> you! </h2>
-            <p class="description-text"> I can help you build a product , feature or website
+        <div class="headline">
+          <div class="headline-container">
+            <div class="card-header">
+              <h2 class="headline-text">I’m Belay <br> Glad to see <br> you! </h2>
+              <p class="description-text"> I can help you build a product , feature or website
                 Look through some of my work and experience! If you
                 like what you see and have a project you need coded,
                 don’t hestiate to contact me.
-            </p>
-            <div>
-                <p class="social-icons-header">Let's Connect</p>
-                <ul class="social-icons">
+              </p>
+            </div>
+              <div class="card-body">
+                <div class="social-liks">
+                  <p class="social-icons-header">Let's Connect</p>
+                  <ul class="social-icons">
                     <li><a href="https://twitter.com/2belamit" aria-label="Twitter" target="_blank" rel="noopener"><i
-                                class="fa-brands fa-twitter"></i></a></li>
-                    <li><a href="https://www.linkedin.com/in/belay-birhanu-144ba714b/" aria-label="Linkedin"
-                            target="_blank" rel="noopener"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                          class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="https://www.linkedin.com/in/belay-birhanu-144ba714b/" aria-label="Linkedin" target="_blank"
+                        rel="noopener"><i class="fa-brands fa-linkedin-in"></i></a></li>
                     <li><a href="https://github.com/belaymit?tab=repositories" aria-label="Github" target="_blank" rel="noopener"><i
-                                class="fa fa-github"></i></a></li>
-                    <li><a href="#" aria-label="Angellist" target="_blank" rel="noopener"><i
-                                class="fa-brands fa-angellist"></i></a></li>
-                </ul>
+                          class="fa fa-github"></i></a></li>
+                    <li><a href="#" aria-label="Angellist" target="_blank" rel="noopener"><i class="fa-brands fa-angellist"></i></a>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+          </div>
+        </div>
+    </header>
+  <main class="main">      
+    <section class="work-section">

```

Comment:
I suggest you put all your cards in this section. I can see that you have put them into different sections however as per the requirements, this are articles and should all be in one section styled using `display: grid`

To make clear, your project should have  3 sections :
- a `headline section` which was the first part of the milestone you did
- A `works section` which contains all the projects you have
- an `about section` which contains the about section

Your portfolio dashboard sections should all be in one section `works section`
Please refer to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) again

 ### Please ensure that each individual article of the work section is styles using flex box to align the items. Its part of the project requirements

<img width="1667" alt="Screenshot 2022-10-28 at 10 52 45" src="https://user-images.githubusercontent.com/48546992/198534542-2c629182-8e66-4b46-ad8e-e162e812d169.png">